server/db/010-addbatteryleveltosessions.sql

```diff
@@ -0,0 +1,21 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 10;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        ALTER TABLE sessions ADD COLUMN button_battery_level integer;
+        ALTER TABLE registry DROP COLUMN button_battery_level;
```

Comment:
Thanks! Could you please make sure that we have an extract of that battery data that's already there so that we have it for buttons which aren't pressed after this change?

I think this can be done anytime now. I don't think there's much benefit in waiting until the deployment since we're only going to get one value from this anyways. What do you think?

Reply:
I put the latest on chatbot-dev and tested it with Postman, feel free to check out. I also copied the registry table to a csv on my machine